src/test/java/io/redis/examples/PipeTransExample.java

```diff
@@ -27,42 +27,49 @@ public void run() {
         // REMOVE_END
 
         // STEP_START basic_pipe
-        AbstractPipeline pipe = jedis.pipelined();
+        // Pipeline is a way to send multiple commands to Redis in a single request.
+        // It can be used to improve performance by reducing the number of round trips to the server.
+        // Make sure to close the pipeline after use to release resources and return the connection to the pool.
```

Comment:
We probably don't need the explanation part here (the code is used in [this page](https://redis.io/docs/latest/develop/clients/jedis/transpipe/), which already has an explanation of pipelines). Also, the width of the code example tab is quite small and we haven't implemented scrolling yet, so it looks a bit neater if we avoid the long comment line.
```suggestion
        // Make sure you close the pipeline after use to release resources
        // and return the connection to the pool.
```